modules/geo-layers/src/mesh-layer/mesh-layer.ts

```diff
@@ -14,12 +22,33 @@ function validateGeometryAttributes(attributes) {
 }
 
 const defaultProps = {
-  // PBR material object. _lighting must be pbr for this to work
   pbrMaterial: {type: 'object', value: null},
   featureIds: {type: 'array', value: null, optional: true}
 };
 
-export default class _MeshLayer extends SimpleMeshLayer {
+/** All properties supported by _MeshLayer. */
+export type _MeshLayerProps<DataT = any> = __MeshLayerProps<DataT> & SimpleMeshLayerProps<DataT>;
+
+/** Properties added by _MeshLayer. */
+type __MeshLayerProps<DataT = any> = {
```

Comment:
Too many underscores... this class is not exposed to the user, we could just name stuff normally in this file.